packages/nimble-components/src/wafer-map/tests/wafer-map-validator.spec.ts

```diff
@@ -78,6 +79,89 @@ describe('Wafermap Validator module', () => {
         });
 
         it('should not be valid', () => {
+            expect(waferMapValidator.getValidity()).toEqual({
```

Comment:
I'll just note that it seems a little strange to have so many describe blocks that just have a single test in them. I see that the whole test file is written this way, so I'm fine with being consistent, I just don't understand why it's in its current form.

Reply:
I think there were many types of inputs to be validated, but the outputs are just is valid or not for this class

Reply:
So, just to make sure there is no confusion with my first remark: I would think most of these tests in this file could be of the form (to use the test this comment is attached to as a specific example):

```
it('with impossible grid dimension wafer map should not be valid', () => {
   const waferMock = getWaferMapMockValidator(1, -1, 1, -1);
   waferMapValidator = new WaferMapValidator(waferMock as WaferMap);
   waferMapValidator.validateGridDimensions();
   
   expect(waferMapValidator.getValidity()).toEqual({
        invalidGridDimensions: true,
        invalidDiesTableSchema: false
    });
    expect(waferMapValidator.isValid()).toBeFalse();
}
````
Now, if there were another test to put under the `with impossible grid dimension` umbrella, this idea would obviously not be good, but the pattern throughout this file suggests that it's intentional to have a single test per `describe` block, which is unconventional.

Reply:
ok, I fixed them